src/MeasureCalculator.java

```diff
@@ -0,0 +1,59 @@
+import java.math.BigDecimal;
+
+public class MeasureCalculator {
+
+    final static int METERS         = 1;
+    final static int CENTIMETERS    = 2;
+    final static int MILIMETERS     = 3;
+    final static int KILOGRAMS      = 1;
+    final static int GRAMS          = 2;
+    final static int MILIGRAMS      = 3;
+
+    private MeasureCalculator(){}
+
+    public static String[] calculateMetric(String inputString, int inputType) {
+        String[] values = new String[3];
+        BigDecimal inputNumber = new BigDecimal(inputString);
```

Comment:
Jak dobrze widzę to nigdzie nie sprawdzasz ani nie obsługujesz sytuacji w której inputString jest np literami a nie liczbami.